Resources/Prototypes/Recipes/Lathes/cooking.yml

```diff
@@ -62,3 +76,24 @@
   completetime: 0.4
   materials:
     Plastic: 50
+
+- type: latheRecipe
+  id: FoodBowlBig
+  result: FoodBowlBig
+  completetime: 0.8
+  materials:
+    Glass: 100
+
+- type: latheRecipe
+  id: FoodPlateTin
+  result: FoodPlateTin
+  completetime: 0.4
+  materials:
+    Steel: 50
```

Comment:
`FoodPlateTin` has a material composition of 100 steel. It'd probably be reasonable to tone that back a bit, but either way, the lathe cost needs to exceed the material composition

Reply:
cromnch